portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,83 @@
-#content {
+/* Website welcome styling */
+#welcome-content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  font-family: 'Questrial', sans-serif;
 }
 
-#greeting-container {
+#fact-container {
   margin-top: 20px;
 }
+
+.button {
+  background-color: #4d0066;
+  border-radius: 50%;
+  color: white;
+  padding: 10px 16px;
+  text-align: center;
+  font-size: 14px;
+}
+
+.logoButton {
+  display: inline-block;
+  height: 40px;
+  width: 40px;
+  margin-right: 10px;
+}
+
+/* Blog styling */
+
+#blog-content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+  font-family: 'Questrial', sans-serif;
+}
+
+/* Gallery styling */
+
+#gallery {
+  margin-left: auto;
+  margin-right: auto;
+  width: 1200px;
+  font-family: 'Questrial', sans-serif;
+}
+
+.row {
+  margin-left: 10px;
+  margin-right: 10px;
+  margin-bottom: 10px;
+}
+
+.row:after {
+  content: '';
+  display: table;
+  clear: both;
+}
+
+.column {
+  float: left;
+  width: 25%;
+}
+
+img.resize {
+  max-width: 80%;
+  max-height: 80%;
+  padding: 2px;
+  border: 3px solid #4d0066;
+}
+
+/* Boxes and borders */
+
+.filledBox {
+  border-radius: 25px;
+  padding: 20px;
+  background: #e6e6ff;
+}
+
+.borderBox {
+  border-radius: 25px;
+  padding: 20px;
+  border: 4px solid #e6e6ff;
+}
```

Comment:
nit: newline at end of file is desirable. :)

This is because if you run `cat <filename>` in linux, you want it to include the new line at the end of the last line so your command prompt shows up on a line of its own.